src/index.js

```diff
@@ -4,9 +4,29 @@ import menu from './menu';
 import contact from './contact';
 
 const parent = document.getElementById('content');
-parent.style.background = "url('assets/img/b.jpg') center";
+parent.style.background = "url('assets/img/b.jpg') center no-repeat";
 
 parent.appendChild(nav());
-parent.appendChild(home());
-parent.appendChild(menu());
-parent.appendChild(contact());
+const link = document.querySelectorAll('.links');
+const main = document.createElement('div');
+
+main.appendChild(home());
+parent.appendChild(main);
```

Comment:
this is what I was talking about when I said modules should be able to append themselves, this logic should be inside of each module (sorry I forgot to point to the lines where it was happening in my review)